SEED-platform/seed
# Refactor/list related data

#### Any background context you want to provide?
SEED is paginating the inventory list page

#### What's this PR do?
- refactors the /properties and /taxlots list and filter endpoints allowing a new query param, `include_related` to indicate whether or not to fetch and include related data (e.g. if fetching properties, whether or not to fetch their linked tax lots).
  - this is backwards compatible, meaning it doesn't break any previous usage which doesn't include the param (it defaults to True, which is how the API previously functioned)
- updated the beta inventory list to _not_ include related data (i.e. you will not see taxlots on the properties table)

#### How should this be manually tested?
- Import some properties and taxlots that are linked, verify that they are still shown in the original table, but are _not_ shown in the beta view

#### What are the relevant tickets?
Closes #3040 
Closes #2902 

#### Screenshots (if appropriate)
Old (with taxlots)
<img width="1083" alt="Screen Shot 2021-12-21 at 5 41 42 PM" src="https://user-images.githubusercontent.com/18518728/147007308-14b2fa3a-29ee-4520-bcbe-b1731c835fea.png">

New (without taxlots)
<img width="1088" alt="Screen Shot 2021-12-21 at 5 41 57 PM" src="https://user-images.githubusercontent.com/18518728/147007334-1ce438c3-1bf0-4ae6-8632-cd1884d49144.png">
